pkg/controller/cloud/nodecontroller.go

```diff
@@ -55,6 +65,13 @@ const (
 
 	// The amount of time the nodecontroller should sleep between retrying NodeStatus updates
 	retrySleepTime = 20 * time.Millisecond
+
+	//Taint denoting that a node needs to be processed by external cloudprovider
+	CloudTaintKey = "ExternalCloudProvider"
```

Comment:
In kubeadm for instance we're using the `node-role.kubernetes.io/master="":NoSchedule` taint for marking the master. The master label is the same (`node-role.kubernetes.io/master=""`)

I really think this should be namespaced in some form.
(nit: `// foo` instead of `//foo`)